Gemfile

```diff
@@ -23,6 +23,9 @@ gem 'jbuilder', '~> 2.0'
 # bundle exec rake doc:rails generates the API under doc/api.
 gem 'sdoc', '~> 0.4.0', group: :doc
 
+gem 'faraday'
+gem 'figaro'
```

Comment:
Nice use of Figaro. The labs don't say anything about it, but I'd been researching how to keep all these API keys secure. Came to the exact same conclusion. =)